tests/base_test_case.py

```diff
@@ -115,9 +115,9 @@ def assertRaisesRegexp(self, expected_exception, expected_regexp,
             self.fail("Expected exception " + str(expected_exception) +
                       " not raised" + msg)
 
-    def assertLazyMessage(self, assert_function, actual, expected, msg_func):
+    def assertLazyMessage(self, msg_func, assert_function, *args, **kwargs):
```

Comment:
I guess that couples the msg_func more tightly with the assertion though.
"All problems in computer science can be solved by another level of indirection"/"...except for the problem of too many layers of indirection." -- Wheeler/Henney